src/lavinmq/amqp/client.cr

```diff
@@ -440,24 +445,60 @@ module LavinMQ
         !@running
       end
 
-      def close_channel(frame : AMQ::Protocol::Frame, code, text)
-        return close_connection(frame, code, text) if frame.channel.zero?
+      enum ChannelReplyCode : UInt16
```

Comment:
I wouldn't mind to put this in its own file. It could be put in AMQP namspace, no need for it to exist under `Client` (so no need for a client directory).

Reply:
👍 
`ChannelReplyCode` and `ConnectionReplyCode` could also maybe be merged to a single `ReplyCode`?

Reply:
Did the first thing, but want to keep them separate to make it clear which replycodes are meant for connection close and channel close.